cypress/integration/team/teammates_pagination_spec.js

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+const uuid = () => Cypress._.random(0, 1e6);
+
+describe('Teams Suite', () => {
+    before(() => {
+        // MaxUsersPerTeam setting must be >60.
+        // Requires more than 60 available test users.
+        // (e.g., make cypress-test-data)
+        Cypress.env('team-size', 60);
+
+        // 1. New test case user
+        cy.loginAsNewUser();
+    });
+
+    it('TS14868 Team Admin can use Next button to page through list in Manage Members', async () => {
+        const job = uuid();
+        const teamDescr = `ts14868-${job}`;
+        const teamKey = `t-${job}`;
+
+        // 2. Create new team for test
+        cy.createNewTeam(teamDescr, teamKey);
+
+        // * Check Town Square channel
+        cy.get('#channelHeaderTitle').should('contain', 'Town Square');
+
+        // 3. Add 60 teammates
+        cy.apiBatchJoinTeam(teamKey, Cypress.env('team-size'));
+
+        // 4. Click hamburger main menu
+        cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+
+        // 5. Click Manage Members
+        cy.get('#sidebarDropdownMenu #manageMembers').should('be.visible').click();
+
+        // * Check Manage Members modal dialog
+        cy.get('#teamMemberModalLabel').should('be.visible');
+
+        // * Check teammate total
+        cy.get('#searchableUserListTotal').should('contain', `1 - 50 members of ${Cypress.env('team-size') + 1} total`);
```

Comment:
I don't think additional 1 is needed here.  Should that only be like below?  Same case for other instance.
```suggestion
        cy.get('#searchableUserListTotal').should('contain', `1 - 50 members of ${teamSize} total`);
```

Reply:
Yeah, this should be set to 60. Test is failing in it's current state. Additionally, I think it'd be valuable to do a count of the actual user entries. Right now each element is in a `div` with a class of `more-modal__row`, so querying for the amount of those should be sufficient.

Reply:
I don't know how to explain it exactly. The behavior I am experiencing is: a) team/_id/members/batch API call succeeds for 20 user items; b) repeating 20 user items registers in the system until 59 total; c) no records past 59 seem to be created. This is more obvious when I experimented by choosing the team size of 70/80. So I put a workaround to call the existing cy.apiAddUserToTeam on each remaining user ID to finish joining team members.